.cspell/library_terms.txt

```diff
@@ -74,6 +74,8 @@ meshgrid
 mimread
 mimsave
 minval
+MMD
```

Comment:
I am 99% sure that this spell check is agnostic to case - if you remove `MMD` does pre-commit fail?

Reply:
Removed it now, pretty sure it's fine.